change_problem.py

```diff
@@ -23,45 +23,63 @@
 matplotlib.use("agg")
 import matplotlib.pyplot as plt
 
+# TODO: Enter your token here
+token = 'YOUR-TOKEN-HERE'
+
 # Set the solver we're going to use
-from dwave.system.samplers import DWaveSampler
-from dwave.system.composites import EmbeddingComposite
+def set_sampler():
+    from dwave.system.samplers import DWaveSampler
+    from dwave.system.composites import EmbeddingComposite
+
+    sampler = EmbeddingComposite(DWaveSampler(endpoint='https://cloud.dwavesys.com/sapi/', token=token, solver={'qpu': True}))
+
+    return sampler
+
+def create_graph():
+    # Create empty graph
+    G = nx.Graph()
+
+    ## TODO:  Update graph to new problem graph
+
+    return G
 
-sampler = EmbeddingComposite(DWaveSampler(solver={'qpu': True}))
+def solve_problem(G, sampler):
 
-# Create empty graph
-G = nx.Graph()
+    ## TODO:  Update dwave-networkx function to new problem function
 
-# Add edges to graph - this also adds the nodes
+    return 
 
-## TODO:  Update graph to new problem graph
+## ------- Main program -------
+if __name__ == "__main__":
 
-# Find the maximum independent set, S
+    G = create_graph()
 
+    sampler = set_sampler()
 
-## TODO:  Update dwave-networkx function to new problem function
+    # Find the maximum independent set, S
+    S = solve_problem(G, sampler)
 
-# Print the solution for the user
-print('Minimum vertex cover size found is', len(S))
-print(S)
+    # Print the solution for the user
+    print('Minimum vertex cover size found is', len(S))
```

Comment:
Whitespace after "is"? `is'` -> `is '`

Otherwise, I am expecting the printout to stick together like `Minimum vertex cover size found is54`.

Reply:
It adds in the whitespace - no additional space is needed.